src/traffic_server/InkAPI.cc

```diff
@@ -1099,7 +1099,7 @@ INKContInternal::destroy()
     // action is needed.
     //
     if (p) {
-      p->schedule_imm(this);
+      p->schedule_imm_local(this);
     }
```

Comment:
I doesn't look to me like this is really changing effective behavior.  schedule_imm() calls schedule(), which will enqueue locally anyway because p == this_ethread():  https://github.com/apache/trafficserver/blob/fef47d7919e0f87e4f650dfb26742c3a0482091b/iocore/eventsystem/P_UnixEThread.h#L96 .

@duke8253  ?

Reply:
Ah, good catch. I originally wrote this on 7.1.x, where that code didn't exist yet. I'll close this PR because it doesn't appear to be useful on 9.0.x and up.